site/assets/styles/scss/styles.scss

```diff
@@ -24,11 +24,69 @@ header {
 }
 
 footer {
+    background-color: #f5f7fa;
     padding: 5rem 6vw;
     width: 100%;
-    color: white;
-    .logo {
-        width: 171px;
+    color: var(--bs-body-color);
+    
+    .ogd-logo {
+        width: 367px;
```

Comment:
Please change this to: 
```
max-width: 367px;
width: 100%;
```
so it scales down in mobile sizes. 
The `header .logo {}` rule also needs similar max-width, width values since that is also too large at mobile sizes, if you don't mind making the change there as well on line 22.